shared/video-sdk/reference/release-notes/web.mdx

```diff
@@ -65,6 +65,14 @@ Chrome plans to deprecate and remove the Plan B dialect in WebRTC. For details,
 </ProductWrapper>
 If you use any of the above SDK versions, Agora recommends that you upgrade the SDK to the latest version.
 
+## v4.14.2
+
+v4.14.2 was released on October 27, 2022.
+
+#### Fixed issues
+
+This release fixed the issue that when using WeChat browsers running on iOS 14.x, the first subscribed audio stream was silent.
```

Comment:
When using WeChat browsers on iOS 14.x, no audio is received from the first channel the user subscribes to.